tutorbackup/plugin.py

```diff
@@ -112,6 +107,8 @@ def backup(context):  # noqa: F811
 
 @k8s_command_group.command(help="restore MySQL, MongoDB, and Caddy")
 @click.pass_obj
+@click.option('--date', default="", type=str,
```

Comment:
One more thought: apparently Click has a `DateTime` type that you can use for options that are meant to be valid dates:

https://www.markhneedham.com/blog/2019/07/29/python-click-date-parameter-type/

Reply:
Oh nice! I'll use that.